src/MahApps.Metro/ThemeManager/ThemeManager.cs

```diff
@@ -937,7 +951,10 @@ public static void SyncThemeWithWindowsAppModeSetting()
                                ? BaseColorLight
                                : BaseColorDark;
 
-            ChangeThemeBaseColor(Application.Current, baseColor);
+            if (Application.Current != null) // In case the Window is hosted in a WinForm app
```

Comment:
I have simply avoided the case where the current app is null but we might want to throw or do something else in this case.

Reply:
We can leave this as it is, because there is already a check at the [HandleUserPreferenceChanged](https://github.com/MahApps/MahApps.Metro/blob/fbc736f286022bf8f2086dbd5f0b16321714d777/src/MahApps.Metro/ThemeManager/ThemeManager.cs#L972-L986) method. The other call is only at the OnStartup of the App itself.